setup.py

```diff
@@ -7,7 +7,7 @@
 import sys
 
 test_suite = "tests"
-tests_require = ["mongo-orchestration>= 0.2, < 0.4", "requests>=2.5.1"]
+tests_require = ["mongo-orchestration>= 0.2, < 0.4", "requests >= 2.5.1", "requests-aws4auth >= 0.7"]
```

Comment:
No need to add these dependencies again in tests_require. They should already be installed (along with mongo-connector, elasticsearch, etc).